src/x509.c

```diff
@@ -12082,20 +12120,35 @@ int wolfSSL_X509_NAME_print_ex(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
         if (len == 0 || buf == NULL)
             return WOLFSSL_FAILURE;
 
-        tmpSz = nameStrSz + len + 3; /* + 3 for '=', comma, and '\0' */
+        tmpSz = nameStrSz + len +
+        #if defined(WOLFSSL_QT)
```

Comment:
we should probably just use the space after comma for all builds that use this function. Since it is a compatibility layer API it should match the formatting that OpenSSL does.

Reply:
Removed Qt guard

Reply:
submit a PR against OpenSSL if you want the output to be different - could take a while to filter down tho ..